### PR TITLE
GraphQL: Remove reference info from Attributes section

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) removes the manually maintained reference information from the topics in the pages/graphql/schema/attributes directory.

## Affected pages

- ...

## Links to Magento Open Source code

- ...

### CODE REVIEW

File: src/pages/graphql/schema/attributes/index.md [hunk 1]
```diff
@@ -3,3 +3,5 @@ title: Attributes
 ---
 
 # Attributes
+
+The attributes queries allow you to retrieve values stored as custom and [Entity-Attribute-Value (EAV)()] attributes. Custom attributes are those added on behalf of a merchant. For example, a merchant might need to add attributes to describe products, such as shape or volume. EAV attributes are defined by the system.
```
Comment (jeff-matthews) [inline src/pages/graphql/schema/attributes/index.md:7]: Should the first occurrence of EAV link somewhere? The formatting looks off; like it's meant to use a link definition, but there is no definition in this file.

File: src/pages/graphql/schema/attributes/queries/custom-attribute-metadata.md [hunk 2]
```diff
@@ -10,10 +10,16 @@ The `customAttributeMetadata` query has been deprecated. Use the [`customAttribu
...
 
+The `StorefrontProperties` output object returns information about a product attribute. Storefront properties are configured in the Admin at **Stores** > Attributes > **Product** > `Attribute Name` > **Storefront Properties**.
+
 ## Syntax
 
 `customAttributeMetadata(attributes: [AttributeInput!]!): CustomAttributeMetadata`
 
+## Reference
+
+The [`customAttributeMetadata`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#query-customAttributeMetadata) reference provides detailed information about the types and fields defined in this query.
```
Comment (jeff-matthews) [inline src/pages/graphql/schema/attributes/queries/custom-attribute-metadata.md:21]: This is the only link that seems to resolve to a specific section.

**Review by jeff-matthews — COMMENTED** [review]
With the exception of the noted URL, none of the other links seem to resolve to a specific reference section. They just take me to the top of the main ref page.

**Comment by keharper** [comment]
`attributesForm`, `atttributesList`, and `customAttributeMetadataV2` are new for 2.4.7, so I've replaced the link so that it points to the Beta version of SpectaQL.

`attributesMetadata` was originally defined in PWA, but unlike the `contactUs` mutation and the `recaptchaV3Config` query, it was not ported to core. I presume it's because the query as been deprecated. I've reverted the contents of this topic. I had already created COMDOX-727 to handle PWA. I thought I was going to be able to close it without any work, because the topics would all be updated in the other sections.

I'll do a little investigating and probably add a sentence stating the query is available only if PWA is installed. I'll do that under COMDOX-727.

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**